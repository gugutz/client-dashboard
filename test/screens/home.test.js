import React from 'react'
import renderer from 'react-test-renderer'
import Home from '../../src/screens/home'

describe('<Home /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Home />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
