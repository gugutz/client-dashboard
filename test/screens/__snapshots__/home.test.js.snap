// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> component should render correctly 1`] = `
<div>
  <h1
    className="title"
  >
    Home
  </h1>
</div>
`;
